Common/gui/guibutton.cpp

```diff
@@ -69,7 +69,7 @@ GUIButton::GUIButton()
 
     IsPushed = false;
     IsMouseOver = false;
-    IsFlipped = false;
+    IsImageFlipped = false;
```

Comment:
My suggestion would actually to make "ImageFlags" here, to ease any possible expansion (and expansion of the save format), since flip may have only limited number of options (H, V, H+V). There are already Flip flags somewhere in the engine (I think you could use ones that Bitmap class uses too).